core/src/com/group66/game/BustaMove.java

```diff
@@ -22,6 +22,21 @@
     /** The logger. */
     public static Logger logger;
     
+    /** Create the only object of this class */
+    private static BustaMove game = new BustaMove();
+   
+    private BustaMove() {   
+    }
+    
+    /**
+     * Gets the only object available.
+     * 
+     * @return the only available object
+     */
+    public static BustaMove getGameInstance() {
+        return game;
+    }
```

Comment:
I think it's nicer to create the instance of the class here if it doesn't exist yet.  Just like in the slides of the lecture. (Design Patterns 2). Make it a synchronized method then, so you make sure there can't be more than 1 instance of it.